orttraining/orttraining/test/python/orttraining_test_lort.py

```diff
@@ -0,0 +1,119 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import unittest
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+# Ask lazy backend to use Pytorch's JIT as
+# lazy backend's executor.
+from torch._lazy.ts_backend import init as init_ts_backend
+
+init_ts_backend()
+
+# Handle ORT dependencies.
+import onnxruntime as ort
+from onnxruntime.capi import _pybind_state as C
+
+# Set up ORT as torch.jit's sub-executor.
+C.register_ort_as_torch_jit_executor()
+
+# Make computation deterministic.
+torch.manual_seed(42)
+ort.set_seed(1)
+
+
+class TestOrtLazyTensor(unittest.TestCase):
+    def test_Simple(self):
+        def run_Simple():
+            # A function to test.
+            def Foo(x):
```

Comment:
Use snake_case for functions; prefer a more meaningful name for the function. e.g. `simple_transform`, etc.?

Reply:
Sure. Will do `test_elementwise_model` and `test_mnist_model`.